swan-lake/integration-tools/openapi-tool.md

```diff
@@ -285,6 +287,70 @@ $ bal openapi -i <openapi-contract> --mode client --client-methods <resource|rem
 
 >**Info:** For more command options, see [OpenAPI to Ballerina CLI options](#openapi-to-ballerina-command-options).
 
+### Automate client generation with Package Build
+
+#### Update the Ballerina.toml file with OpenAPI tool configurations
+
+Adding the following OpenAPI tool configurations to the Ballerina.toml file will generate a client during the `bal build`. This eliminates the need to commit the generated code.
+
+> **Info:** It is mandatory to provide `id`,` filePath`, and `targetModule` attributes for the OpenAPI tool configurations. The other attributes are optional.
```

Comment:
The `targetModule` is also optional and defaulted to the `default` module. As a best practice we need to provide the `targetModule` otherwise it may have some conflicts to the files which are in the `default` module (outside the generated directory). Is that the reason for mentioning it as a `mandatory` option?

Reply:
will change this to optional.